docs/guides/general/passing_parameters.md

```diff
@@ -0,0 +1,153 @@
+
+- Parameters of a **:octicons-package-dependents-24: task** in [`dbx execute`](../../reference/cli#dbx-execute)
+- Parameters of a **:octicons-workflow-24: workflow** in the [asset-based launch mode](../../features/assets.md) (`dbx launch --from-assets`)
+- Parameters of a **:octicons-workflow-24: workflow** in the normal launch mode (`dbx launch`)
+
+All parameters should be provided in a form of a JSON-compatible payload (see examples below).
+
+!!! tip "Multiline strings in the shell"
+
+    To handle a multiline string in the shell, use single quotes (`'`), for instance:
+
+    ```bash
+    dbx ... --parameters='{
+      "nicely": "formatted"
+      "multiline": [
+        "string",
+        "with",
+        "many",
+        "rows"
+      ]
+    }'
+    ```
+
+=== ":material-lightning-bolt-circle: `dbx execute`"
+
+    The `execute` command expects the parameters only for one specific task (if it's a multitask job) or only for the job itself (if it's the old 2.0 API format).
+    The provided payload shall match the expected payload of a chosen workflow or task.
+    It also should be wrapped as a JSON-compatible string with curly brackets around the API-compatible payload.
+    Since `dbx execute` only supports `spark_python_task` and `python_wheel_task`, only the compatible parameters would work.
+
+    Examples:
+    ```bash
+    dbx execute <workflow_name> --parameters='{"parameters": ["argument1", "argument2"]}' # compatible with spark_python_task and python_wheel_task
+    dbx execute <workflow_name> --parameters='{"named_parameters": ["--a=1","--b=2"]}' # compatible only with python_wheel_task
+    ```
+
+=== ":material-lightning-bolt-outline: `dbx launch --assets-only`"
+
+    !!! tip "General note about 2.0 and 2.1 Jobs API"
+
+        We strongly recommend using the Jobs API 2.1, since it provides extensible choice of tasks and options.
+
+    Assets-based launch provides 2 different interfaces depending on the Jobs API version you use.
+
+    For Jobs API 2.0, the payload format should be compliant with the [RunSubmit](https://docs.databricks.com/dev-tools/api/2.0/jobs.html#runs-submit) API structures in terms of the task arguments.
+
+    Examples for Jobs API 2.0:
+    ```bash
+    dbx launch <workflow_name> --assets-only --parameters='{"base_parameters": {"key1": "value1", "key2": "value2"}}' # for notebook_task
+    dbx launch <workflow_name> --assets-only --parameters='{"parameters": ["argument1", "argument2"]}' # for spark_jar_task, spark_python_task and spark_submit_task
+    ```
+
+    For Jobs API 2.1, the payload format should be compliant with the [JobsRunSubmit](https://docs.databricks.com/dev-tools/api/latest/jobs.html#operation/JobsRunsSubmit).
+    This API supports task-level parameter passing.
+
+    Examples for Jobs API 2.1:
+
+    ```bash
+    # notebook_task
+    dbx launch <workflow_name> --assets-only <workflow_name> --parameters='[
+        {"task_key": "some", "base_parameters": {"a": 1, "b": 2}}
+    ]'
```

Comment:
Just wanted to add a comment to this documentation. I'm using dbx 0.7.6 with Jobs API 2.1. I found this only worked with a command like:
```bash
dbx launch <workflow_name> --from-assets --parameters='{
    "tasks": [
        {"task_key": "some",
         "notebook_task": {"base_parameters": {"a": 1, "b": 2}}
        }
    ]
}
```
As a side-note, `dbx launch <workflow-name> --assets-only` raises an error and suggests using `--from-assets` instead. Is this a typo, or a change to expect in the yet-to-be-released dbx 0.8.0?

Reply:
As it's stated in documentation - this will work correctly since 0.8.0